tests/inference_runner.py

```diff
@@ -1,10 +1,36 @@
-from frontend.stmt_inferrer import *
+from frontend.pre_analysis import PreAnalyzer
+import frontend.z3_types as z3_types
+import ast
 
-r = open("tests/expressions_test.py")
+r = open("tests/inference/classes_test.py")
 t = ast.parse(r.read())
+
+analyzer = PreAnalyzer(t)
+
+classes_data = analyzer.classes_pre_analysis()
+z3_types.init_types({
+    "max_tuple_length": analyzer.maximum_tuple_length(),
+    "max_function_args": analyzer.maximum_function_args(),
+    "classes_to_attrs": classes_data[0],
+    "class_to_base": classes_data[1]
+})
+
+from frontend.stmt_inferrer import *
```

Comment:
Because I wanted the pre-analyser to config the types before importing the stmt_inferrer. Otherwise, the stmt_inferrer module will use non configured types.